forge-gui/res/cardsfolder/upcoming/conspiracy_unraveler.txt

```diff
@@ -0,0 +1,8 @@
+Name:Conspiracy Unraveler
+ManaCost:5 U U
+Types:Creature Sphinx Detective
+PT:6/6
+K:flying
+S:Mode$ Continuous | Affected$ Card.YouCtrl+nonLand | AddKeyword$ Alternative Cost:CollectEvidence<10> | AffectedZone$ Hand,Graveyard,Exile,Library,Command | Description$ You may collect evidence 10 rather than pay the mana cost for spells that you cast. (To collect evidence 10, exile cards with total mana value 10 or greater from your graveyard.)
```

Comment:
would this be better as a MayPlayAltManaCost ?

Reply:
Can't get it to work with MayPlayAltManaCost, any thoughts ?